.github/README.md

```diff
@@ -0,0 +1,166 @@
+## What are GitHub Actions?
+
+GitHub Actions are used for many different purposes.  We use them to run tests in CI, validate PRs are in an expected state, and automate processes.
+
+- [Overview of GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions)
+- [What's a workflow?](https://docs.github.com/en/actions/using-workflows/about-workflows)
+- [GitHub Actions guides](https://docs.github.com/en/actions/guides)
+
+## dbt Labs Action Repo
+[dbt-labs/actions](https://github.com/dbt-labs/actions/) is a central repository of actions and workflows dbt uses across repositories.
+
+## General Standards
+
+### Permissions
+- Default to least permissions.
```

Comment:
You also might want to state what the default is which is 
`Workflows have read permissions in the repository for the contents scope only.`

FYI- this is something I'm setting at the repo levels today- I just saw the toggle to do this so we might have to fix some actions that don't abide by this currently